confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/pipe/task/CreatePipeProcedureV2.java

```diff
@@ -178,16 +178,15 @@ protected void rollbackFromWriteConfigNodeConsensus(ConfigNodeProcedureEnv env)
   }
 
   @Override
-  protected void rollbackFromOperateOnDataNodes(ConfigNodeProcedureEnv env) {
+  protected void rollbackFromOperateOnDataNodes(ConfigNodeProcedureEnv env) throws IOException {
```

Comment:
However it may still be 2 interfaces, not 1. If we somehow represent the similarity in 2 functions by squashing them into 1 function, the independence of their meanings may be hurt.

Reply:
However it may still be 2 interfaces, not 1. If we somehow represent the similarity in 2 functions by squashing them into 1 function, the independence of their meanings may be hurt.